cmd/zc_traverser_s3.go

```diff
@@ -122,6 +134,13 @@ func (t *s3Traverser) traverse(preprocessor objectMorpher, processor objectProce
 			continue
 		}
 
+		if invalidAzureBlobName(objectInfo.Key) {
+			//Throw a warning on console and continue
+			msg := fmt.Sprintf("Skipping S3 object %s, as it is not a valid Blob name. Rename the object and retry the transfer", objectInfo.Key)
```

Comment:
Log this into the scanning logger as well.

Reply:
Add this to scanner logs.